engine.js

```diff
@@ -203,6 +177,13 @@ HMDAEngine.prototype.getFileSpec = function(year) {
     return hmdaRuleSpec.getFileSpec(year);
 };
 
+/**
+ * Get the progress object used for task completion events displayed on the progress bar
+ * @return {object} Progress object containing an eventemitter for progress notification
+ */
+HMDAEngine.prototype.getFileProgress = function() {
+    return hmdajson.getProgress();
+};
```

Comment:
Ah ok, I see what you did now. Shouldn't there be a similar convenience function for getting the ruleProgress or for clearing the progress?